### PR TITLE
Add app layout

### PR DESCRIPTION
**Pull request by manusmd**
# What I did
- Used self-made boilerplate as base
- Added CSS Variables
- Designed app layout  
# Design
rickmorty

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+  --font-card: 'Space Grotesk', sans-serif;
+}
+
+body {
+  font-family: var(--font-primary);
+  margin: 0;
+  background-color: var(--color-background);
+}
+
+.header {
```
Comment (davidwobido) [inline style.css:26]: Don’t need the dot for a class for header
```suggestion
header {
```

File: style.css [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.rickmortyIMG {
+  height: 8rem;
+  margin: 0.5rem;
+}
+
+.main {
```
Comment (davidwobido) [inline style.css:41]: Same as header, the dot is not necessary
```suggestion
main {
```

**Review by DennisTarik — COMMENTED** [review]
can you please reply to my question in slack?
it looks good to me otherwise

File: style.css (hunk 1 above)
Comment (lmachens) [reply style.css:26]: I don't agree, because `header` could be used multiple times

**Pull request merged**